amida-tech/hl7-telescoper-api-server
# Hl7 2 ajp upload file to server

Ticket: https://jira.amida.com/browse/HL7-2

## What does this ticket do?
- Adds endpoint to upload a file (using multer)
- Moves sample HL7 messages to `/data` folder
- Removes JWT requirement from config file
- Adds unit test for upload function
- Note that this doesn't do any validation of the file that is uploaded, that is a separate [ticket](https://jira.amida.com/browse/HL7-11). This only uploads the file and saves it. It was mainly to create a template so other people can work around it. 

## How to Test:
 -  Send a POST request from postman with a file in the request body
 - Make sure the file gets uploaded to `data/hl7-uploads`
 - Make sure your mongoDB has the name of the file and the date/time it was uploaded
- Make sure `yarn test` passes and there are no linting errors.
